extensions/Webhooks/template/en/default/account/prefs/webhooks.html.tmpl

```diff
@@ -64,8 +80,7 @@ YAHOO.util.Event.onDOMReady(onRemoveChange);
 </script>
 
 <p>
-  It will be sent a POST request with all the information of the bugs that match with the events and filters selected to your URL.<br>
-  Documentation about webhooks is available <a href="">here</a>.
+  It will be sent a POST request with the information of the bugs that match with the events and filters selected to your URL.
```

Comment:
It will be sent a POST request with the information of the [% terms.bugs %] that match with the events and filters selected to your URL.